addons/sourcemod/scripting/shavit-mapchooser.sp

```diff
@@ -758,10 +769,14 @@ void InitiateMapVote(MapChange when)
 	if ((when == MapChange_MapEnd && add_extend))
 	{
 		menu.AddItem("extend", "Extend Current Map");
+		menu.AddItem("reroll", "Reroll Maps");
+		instantChange = false;
 	}
 	else if (when == MapChange_Instant)
 	{
 		menu.AddItem("dontchange", "Don't Change");
+		menu.AddItem("reroll", "Reroll Maps");
```

Comment:
Reroll should be before extend/dontchange because this is destroying people's muscle memory